src/content/en/ilt/pwa/lab-workbox.md

```diff
@@ -66,28 +66,41 @@ Navigate to the __project__ directory via the command line:
 Run the following command to install the project dependencies:
 
     npm install
-    npm install gulp --global
 
-Then build and serve the app with the gulp `serve` task:
+Then build and serve the app with the following command:
 
-    gulp serve
+    npm run serve
 
-You can terminate the `gulp serve` process at any time with `Ctrl-c`.
+You can terminate the `serve` process at any time with `Ctrl-c`.
 
 #### Explanation
 
 The `npm install` command installs the project dependencies based on the configuration in __package.json__. Open __project/package.json__ and examine its contents.
 
 The important package is  [`workbox-build`](/web/tools/workbox/guides/precache-files/workbox-build), which is a module used to generate a list of assets that should be precached in a service worker. You'll see how that's used in a later step.
 
+Also note that the `serve` script aliases the `gulp serve` command.
+
 The remaining dependencies in __package.json__ are used by gulp in __gulpfile.js__.
 
-Open __gulpfile.js__ and examine its contents. The `gulp serve` command runs the `serve` task. The `serve` task "builds" the app; in this case by copying the __src__ files into a __build__ directory. The __build__ directory is then served and `watch`ed. The `watch` functionality ensures that the app is rebuilt whenever __src__ files are updated.
+Open __gulpfile.js__ and examine its contents. The gulp `build` task "builds" the app; in this case by copying the __src__ files into a __build__ directory (using the `clean` and `copy` tasks). The `serve` task builds the app (with the build task) and then the __build__ directory served and `watch`ed. The `watch` functionality ensures that the app is rebuilt whenever __src__ files are updated.
```

Comment:
This needs a verb as shown:
'directory is served and watched. '

Mixing code with non code as you did with `watch`ed is not a best practice. Try rewriting to avoid.